### DISCUSSION TITLE
Feature Request: Set dependencies between pull requests...

### DISCUSSION THREAD

**Discussion by DennisHeimbigner** [community/community#40571] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Our organization often has to create pull requests that need to be merged
in a specific order. It would be convenient if it was possible to indicate that
a pull request could not be committed until some other set of pull requests
were committed.

**Comment by ossanna16** [comment]
Hi there @DennisHeimbigner and welcome to our community! Thank you for the feedback!

**Comment by bkuberek** [comment]
Hi there, I was looking for the same thing.

I would like to see the ability to block pull request merges based on other pull requests that are open, creating a dependency between two or more pull requests. Perhaps we can leverage checks for that?

For example, project A has a library that needs to be updated before project B can merge the service that uses such library. In this scenario, Project B / pull request B1, has a dependency on project A / pull request A1. A1 blocks B1 from being merged until A1 is merged.

Ideally this would be available in Github Enterprise also.
